pkg/scheduler/internal/cache/debugger/comparer.go

```diff
@@ -57,11 +57,11 @@ func (c *CacheComparer) Compare() error {
 	pendingPods := c.PodQueue.PendingPods()
 
 	if missed, redundant := c.CompareNodes(nodes, dump.Nodes); len(missed)+len(redundant) != 0 {
-		klog.Warningf("cache mismatch: missed nodes: %s; redundant nodes: %s", missed, redundant)
+		klog.InfoS("Cache mismatch", "missedNodes", missed, "redundantNodes", redundant)
```

Comment:
I cannot get the point of change "missed nodes" to "missedNodes", what's the consideration for this?